docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,242 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/swarm/manager-administration-guide/"
+]
+title = "Swarm Manager Administration Guide"
+description = "Manager administration guide"
+keywords = ["docker, container, cluster, swarm, manager, raft"]
+advisory = "rc"
+[menu.main]
+identifier="manager_admin_guide"
+parent="engine_swarm"
+weight="12"
++++
+<![end-metadata]-->
+
+# Administer and maintain a swarm of Docker Engines
+
+When you run a swarm of Docker Engines, **manager nodes** are the key components
+for managing the cluster and storing the cluster state. It is important to understand
+some key features of manager nodes in order to properly deploy and maintain the
+swarm.
+
+This article covers the following swarm administration tasks:
+
+* [Add Manager nodes for fault tolerance](#add-manager-nodes-for-fault-tolerance)
+* [Distributing manager nodes](#distributing-manager-nodes)
+* [Running manager-only nodes](#run-manager-only-nodes)
+* [Backing up the cluster state](#back-up-the-cluster-state)
+* [Monitoring the swarm health](#monitor-swarm-health)
+* [Recovering from disaster](#recover-from-disaster)
+
+Refer to [How swarm mode nodes work](how-swarm-mode-works/nodes.md)
+for a brief overview of Docker Swarm mode and the difference between manager and
+worker nodes.
+
+## Operating manager nodes in a swarm
+
+Swarm manager nodes use the [Raft Consensus Algorithm](raft.md) to manage the
+cluster state. You only need to understand some general concepts of Raft in
+order to manage a swarm.
+
+There is no limit to scale for manager nodes. The decision about how many
```

Comment:
Let's say "There is no limit on the number of manager nodes". The current phrasing makes it sound like adding more manager nodes is a way to scale the cluster.